samples/snippets/dataset_access_test.py

```diff
@@ -22,6 +22,7 @@
     import pytest
 
 
+# is it possible to cleanup these datasets if they are passed in?
```

Comment:
There may be a way, while we are examining and testing this process, to use the `capsys` module in pytest to capture outputs to stdout/stderr and examine them (ie. as a list of items that got deleted OR something).

See line 38 below.

https://docs.pytest.org/en/7.1.x/reference/reference.html#capsys

https://docs.pytest.org/en/7.1.x/_modules/_pytest/capture.html

Reply:
Will be removing this comment as I am prioritizing other workloads